Bela/main.cpp

```diff
@@ -223,8 +292,8 @@ bool csound_setup(BelaContext *context, void *p)
   CsData *csData = (CsData *) p;
   Csound *csound;
   const char *args[] = { "csound", csData->csdfile.c_str(), "-iadc",
-			 "-odac", "-+rtaudio=null",
-			 "--realtime", "--daemon" };
+       "-odac", "-+rtaudio=null",
```

Comment:
are these (and other) whitespace changes expected? they make the diff excessively large. If these are intended (e.g.: to conform to code standards), I would recommend to have them all in a dedicated commit.

Reply:
Not really - must have happened when copying the original main.cpp code from Sublime to the Bela IDE and back. I'll try to revert them before updating or making a new PR